definitions/json/CaseTypeTab/CaseTypeTab.json

```diff
@@ -1036,21 +1036,11 @@
     "CaseFieldID": "applicationsAdminLinks",
     "TabFieldDisplayOrder": 3
   },
-  {
-    "CaseFieldID": "genericTseApplicationCollection",
```

Comment:
This shouldn't be removed from the tab as then the caseworker has no way of seeing any application

Reply:
restored